apistar/backends/sqlalchemy.py

```diff
@@ -16,7 +16,7 @@ def build(cls, settings: Settings):
         metadata = config.get('METADATA')
 
         kwargs = {}
-        if url.startswith('postgresql:'):  # pragma: no cover
+        if url.startswith('postgresql'):  # pragma: no cover
```

Comment:
This probably wants to be `if not url.startswith('sqlite'):` as mysql may also want to set the pool size